### PR TITLE
Create machine_learning_in_r.Rmd

### PR DESCRIPTION
**Pull request by KenSu223**
Adding R machine learning workflow tutorial

### CODE REVIEW

File: machine_learning_in_r.Rmd [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+title: "R Machine Learning Workflow Tutorial"
+author: "Tongli Su(ts3478), Yifan Lu(yl5113)"
+output:
+  html_document:
+    fig_width: 5
+    fig_height: 3.5
+  pdf_document: default
+---
```
Comment (naishadhparmar) [inline machine_learning_in_r.Rmd:9]: Hi, please remove this YAML header and replace it with a header starting with a single hashtag and a single space, followed by a blank line, followed by the author's name.

image

**Review by naishadhparmar — CHANGES_REQUESTED** [review]

**Comment by KenSu223** [comment]
Thanks for the comment. I deleted the old file due to a read-only permission issue, and uploaded a new file with the correct header.

**Comment by jtr13** [comment]
@naishadhparmar ls this ready to be merged now?

**Comment by naishadhparmar** [comment]
Hi, sorry I missed this. It seems like another group has made a community contribution with the same file name, leading to merge conflicts. Can we please change the name of this file to something else?

**Comment by KenSu223** [comment]
@jtr13 Hi Professor. It seems like our branch is still not merged. Could you please help merge it to the main? Thanks!

**Comment by jtr13** [comment]
Thanks for your contribution and apologies for the delay in merging! 🎉

**Pull request merged**